DFC.Api.Content/Models/Cypher/GenericCypherQuery.cs

```diff
@@ -0,0 +1,39 @@
+﻿using DFC.ServiceTaxonomy.Neo4j.Queries.Interfaces;
+using Neo4j.Driver;
+using System.Collections.Generic;
+using DFC.ServiceTaxonomy.Neo4j.Queries;
+
+namespace DFC.Api.Content.Models.Cypher
+{
+    public class GenericCypherQuery : IQuery<IRecord>
```

Comment:
It'd be good to have a generic Cypher query in the Neo4j package and use that one here.
Not sure if we can have a single one, although it'd be nice (noticed this version is slightly different to the one in the editor).
We don't need to make the change for the initial commit though - we can do it later.